settings/DevHome.Settings/Views/AccountsPage.xaml.cs

```diff
@@ -145,7 +145,7 @@ private async Task ConfigureLoginUIRenderer(AdaptiveCardRenderer renderer)
         }
         catch (Exception ex)
         {
-            GlobalLog.Logger?.ReportError($"Failure occurred while retrieving the HostConfig file - Error: {ex} HostConfigFileName: {hostConfigFileName}");
+            GlobalLog.Logger?.ReportError($"Failure occurred while retrieving the HostConfig file -- HostConfigFileName: {hostConfigFileName}.", ex);

```

Comment:
NIT: why the extra hyphen?

Reply:
Can't remember why; seems like no reason. Reverted.